src/widget/extra_info.h

```diff
@@ -0,0 +1,28 @@
+#ifndef WIDGET_EXTRA_INFO_H
```

Comment:
I'd rather call this `sidebar_filler`, and use it for drawing both the extra info and the relief images below it.

The interface can then become much simpler:
- one function for drawing the collapsed filler, arguments: x, y, width, height
- one function for drawing the expanded filler, arguments: x, y, width, height
- one function for handling mouse (the button clicks), arguments: mouse, x, y, width, height

Reply:
ok, sounds good - ill make some open issues for these functions, as i want this pr to do as little logic changes as possible.  ill tackle em after the new files are safely in master

Reply:
Actually, please do it in this PR, since I'm not going to merge this one with the global variables and the non-conforming naming convention still in here.
And please don't create issues for code cleanup, just submit PRs. I'd like to keep the issues for, well, issues related to gameplay etc.

Reply:
ok - renamed and opened a new issue https://github.com/bvschaik/julius/issues/425

Reply:
ok, will do, ill close out the issue i made